tests/unit/unit_tests.hpp

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2010-2020, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#ifndef MFEM_UNIT_TEST
+#define MFEM_UNIT_TEST
+
+#include "catch.hpp"
+
+// MFEM_Approx can be used to compare floating point values within an absolute
+// tolerance of `margin` (default value 1e-12).
```

Comment:
```suggestion
/// MFEM_Approx can be used to compare floating point values within an absolute
/// tolerance of `margin` (default value 1e-12).
```

Reply:
Do you know if Doxygen documentation is generated for the unit tests? My impression was that it's not. Not that the extra `/` hurts...

Reply:
I'm not sure about Doxygen; I only noticed that when I try reading the function documentation in XCode, then it only reads it correctly when using `///`; otherwise, if using `//`, it won't display any information about the function at all. The XCode project files are generated via CMake.

Reply:
Oh, I didn't realize this also affected Xcode documentation. Good to know!